README.md

```diff
@@ -1,6 +1,6 @@
 # Solidus Internationalization
 
-[![Build Status](https://travis-ci.org/solidusio/solidus_i18n.svg?branch=master)](https://travis-ci.org/solidusio/solidus_i18n)
+[![CircleCI](https://circleci.com/gh/solidusio/solidus_i18n.svg?style=svg)](https://circleci.com/gh/solidusio/solidus_i18n)
```

Comment:
Maybe there's still some problem with this svg... I see a `no builds` icon [here], (https://github.com/aldesantis/solidus_i18n/tree/circleci) 🤔 

Also, it seems Code Climate is not active anymore for this repo? But this can be fixed later of course.

Reply:
@spaghetticode this is because the repo needs to be activated in CircleCI by @kennyadsl (not sure if you can also take care of it?). Once that is done, I can push an empty commit to run CircleCI on this PR and ensure everything's in order. Then we can merge the PR and adjust the required PR checks.

Reply:
And yes, Code Climate would also need to be removed, but I think we can do that separately as you said. :)